### PR TITLE
Add note explaining realtime updates websockets setup with HTTPS

### PR DESCRIPTION
**Pull request by ajwild**
Relations:
  - Issue: Fixes https://github.com/livingdocsIO/livingdocs-bugs/issues/5104

### CODE REVIEW

File: content/customising/server-configuration/_index.md [hunk 1]
```diff
@@ -617,7 +617,12 @@ Since {{< release "release-2023-11" >}} the `realtimeUpdates` object has been ex
...
 
-#### Document-lists
+If you are running the Livingdocs server locally using HTTPS with a self-signed certificate then you will need to start the editor with `NODE_TLS_REJECT_UNAUTHORIZED=0` to establish the websocket connection. This could either prefix the npm start script, be set or exported in the terminal, be added before the start command executed in the terminal, or defined in the runtime code by modifying `process.env`. Please be aware that this is for local development only and should not be needed or used in production. You should see the following warning displayed when node starts if it was applied correctly:
```
Comment (marcbachmann) [inline content/customising/server-configuration/_index.md:620]: is this only the case for websockets? doesn't the other http requests also fail?

**Review by marcbachmann — APPROVED** [review]

File: content/customising/server-configuration/_index.md (hunk 1 above)
Comment (ajwild) [reply content/customising/server-configuration/_index.md:620]: "Normal" requests succeed without this. When you open the editor and accept the self-signed certificate for the session I guess that sets it as valid for all HTTPS requests, but not for WSS requests.

**Comment by ajwild** [comment]
@livingdocsIO/team-customer-review please could you check if the instructions here make sense for the downstreams that had this problem.

**Comment by meinradandermatt** [comment]
@ajwild It works fine for me. Now I can also see the Admin indexing page. 👍

**Review by meinradandermatt — APPROVED** [review]

**Pull request merged**